README.md

```diff
@@ -56,6 +58,16 @@ Stacktrace:
 This is because `stdout` and `stderr` within an `iocapture` actually refer to the temporary
 redirect streams which get cleaned up at the end of the `iocapture` call.
 
+### ANSI color/escape code
+
+On Julia v1.6 and later, if the `color` keyword argument is set to `true`, the captured
+output of `iocapture` inherits the `:color` property of the `stdout` or `stderr`.
+
+On the other hand, on Julia v1.5 or earlier, even if the `color` keyword argument is set to
+`true`, no coloring will be applied. However, this limitation might be removed in the
+future.
```

Comment:
```suggestion
On Julia 1.5 and earlier, setting `color` to `true` has no effect, because the [ability to set `IOContext` attributes on
redirected streams was added in 1.6](https://github.com/JuliaLang/julia/pull/36688). I.e. on those older Julia versions
the captured output will usually never contain the ANSI color escape sequences.
```

Reply:
I think your suggestion is a good improvement.
One concern, however, is that the `color=true` of having no effect is only an implementation matter. For example, we are "technically" capable of implementing pipe-ends with i/o contexts.

Of course, we are nearing the release of v1.6, and given that v1.6 is the next LTS candidate, I don't think that the behavior on v1.5 or earlier will change.
So, if there are no other opinions, I would like to commit your suggestion.

Reply:
The suggestion is mainly to rephrase it so it would focus more on the limitation of the implementation (since it's under the "Known limitations") section. If we, at some point, implement color support in <=1.5, we'd just remove this paragraph, as it's would not apply anymore.

Reply:
Ah, I read that paragraph as implying "won't fix". :sweat_smile: